cspell.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.2"
+language: en
+allowCompoundWords: true
+dictionaries:
+  - node
+  - typescript
+words:
+  - autorest
+  - azsdkengsys
+  - azurecr
+  - blockful
+  - blockless
+  - cadl
+  - Contoso
+  - CRUDL
+  - dogfood
+  - eastus
+  - esbuild
+  - globby
+  - inmemory
+  - instanceid
+  - intrinsics
+  - jsyaml
+  - msbuild
+  - MSRC
+  - munge
+  - mylib
+  - nostdlib
+  - oapi
+  - oneof
+  - onig
+  - onigasm
+  - onwarn
+  - openapi
+  - openapiv
+  - picocolors
+  - proto
+  - protobuf
+  - protoc
+  - regen
+  - rpaas
+  - rushx
+  - safeint
+  - strs
+  - TSES
+  - Uhoh
+  - unassignable
+  - Uncapitalize
+  - uninstantiated
+  - unioned
+  - unprojected
+  - unsourced
+  - unversioned
+  - vsix
+  - vswhere
+  - Vtbzpw
```

Comment:
?

Reply:
its for those samples  Authorization: Basic ZGVtbzpwQDU1dzByZA==.

There was 3 locations for that. Which felt having a bit too much inline suppression

Reply:
But maybe I can have another ignore pattern `/.*==/g`